x-pack/plugins/endpoint/server/routes/index.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IRouter, KibanaResponseFactory } from 'kibana/server';
+
+export function addRoutes(router: IRouter) {
+  router.get(
+    {
+      path: '/endpoint/hello-world',
+      validate: false,
+    },
+    greetingIndex
+  );
+}
+
+async function greetingIndex(...passedArgs: [unknown, unknown, KibanaResponseFactory]) {
```

Comment:
Let's use `RequestHandler` as a more descriptive type. We will remove the required types from the generic soon.
```ts
import { RequestHandler } from 'src/core/server';
export const greetingIndex: RequestHandler<any, any, any> = async (context, request, response) => {
```